### PR TITLE
Remove legacy girder_worker code

### PR DESCRIPTION
**Pull request by kotfic**
Still a work in progress

### CODE REVIEW

File: requirements.in [hunk 1]
```diff
@@ -1,4 +1,4 @@
-celery>=4,<4.2
+celery
 funcsigs ; python_version < '3.3'
 girder-client>=2
 networkx<2
```
Comment (danlamanna) [inline requirements.in]: This dependency shouldn't be required anymore

**Comment by danlamanna** [comment]
I can't approve it because the diff crashes my browser. +1

**Comment by zachmullen** [comment]
So this is good to go? Should I change the required status checks?

**Comment by zachmullen** [comment]
Nevermind, looks like someone took care of that.

**Pull request merged**